content/en/blog/releases/announcing-opencue-v0.1.91-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Announcing the release of OpenCue v0.1.91"
+linkTitle: "Version v0.1.91 release"
+date: 2019-02-27
+description: "OpenCue v0.1.91 release notes"
+---
+
+  v0.1.91 of OpenCue includes the following changes and updates:
+
+ *   Cuebot writes warn and error messages to stdout [#191](https://github.com/imageworks/OpenCue/pull/191).
+*   Fixed submitting dependent jobs from PyOutline [#192](https://github.com/imageworks/OpenCue/pull/192).
+*   CueSubmit dialog is now resizable and scrollable [#205](https://github.com/imageworks/OpenCue/pull/205).
+*   Server error messages are now sent to clients making API requests [#59](https://github.com/imageworks/OpenCue/pull/59).

```

Comment:
Should we break up these long lines for readability, like in g3doc?

Reply:
+1 and done.